src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,17 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.PasswordValidationException;
+
 public class UserService {
     public void registerUser(User user) {
+        PasswordValidator passwordValidator = new PasswordValidator();
 
+        try {
+            passwordValidator.validate(user.getPassword(), user.getRepeatPassword());
+            saveUser(user);
+        } catch (PasswordValidationException | NullPointerException e) {
```

Comment:
catching RuntimeExceptions (NullPointerException in your case) is bad practice - please remove it from catch block

Reply:
done